web_transport/impl/quic_transport_stream_impl.h

```diff
@@ -27,7 +28,7 @@ namespace quic {
 // QuicTransportStreamImpl is a proxy for ::quic::QuicTransportStream. All calls
 // to ::quic::QuicTransportStream run in runner_.
 class QuicTransportStreamImpl : public QuicTransportStreamInterface,
```

Comment:
Are you planning to rename interfaces to WebTransportXXX as well in this PR?

Reply:
No, this PR only fix compiling issues after rebasing Chromium to 93. GN targets and directory name have been renamed to web-transport, but the implementation is still quic-transport.